openrarity/providers/opensea/client.py

```diff
@@ -177,7 +210,29 @@ async def fetch_opensea_assets_data(
     before_sleep=before_sleep.before_sleep_log(logger, logging.WARN),
 )
 async def _send_request(client: "httpx.AsyncClient", url: str, params: dict[str, Any]):
+    """Sends httpx.asyncclient request to get token assests data.
+
+    Parameters
+    ----------
+    client: "httpx.AsyncClient"
+        HTTPX async client.
+    url: str
+        Https url to get the data.
```

Comment:
Doesn't have to be HTTPS. It can be regular HTTP as well. But something like 
> URL to submit GET request with provided query args against. 
is fine